README.md

```diff
@@ -2,45 +2,41 @@
 
 # Currencycloud API v2 Java client
 
-## Version: 0.7.8
+## Version: 0.9.1
 
-This is the official Java SDK for the Currencycloud API. Additional documentation 
-for each API endpoint can be found at [developer.currencycloud.com][docs]. 
+This is the official Java SDK for the Currencycloud API. Additional documentation for each API endpoint can be found at [developer.currencycloud.com][docs]
 
-If you have any queries or you require support, please contact our sales team at sales@currencycloud.com.  Please quote your login id in any correspondence as this makes
-it far simpler for us to locate your account and give you the support you need.
+If you have any queries or you require support, please contact our development team at development@currencycloud.com Please quote your login id in any correspondence as this makes it easier for us to locate your account and give you the support you need.
 
 ## Prerequisites
 
 ### 1. Maven (optional, but highly recommended)
 
-CurrencyCloud-Java is a Maven project.  We highly recommend using [Apache Maven][maven] 3 (or a compatible build tool like Gradle) 
-to build your project.  While using Maven is not strictly required, 
-it will be far easier to simply make use of Maven to build the project and do the dependency management for you.
+CurrencyCloud-Java is a Maven project. We highly recommend using [Apache Maven][maven] 3 (or a compatible build tool like Gradle) 
+to build your project. While using Maven is not strictly required 
+it will simplify building the project and handling dependencies.
 
+### 2. Oracle JDK 8 or equivalent JDK
 
-### 2. Oracle JDK 7 or equivalent JDK
-
-CurrencyCloud-Java requires at least a Java version 7 compatible JDK.
+CurrencyCloud-Java requires at least a Java version 8 compatible JDK.
 
 ### 3. A valid sandbox login id and api key on the Currencycloud sandbox API environment.
 
-You can register for demo API key at [developer.currencycloud.com][developer]. 
+You can register for a demo API key at [developer.currencycloud.com][developer]
 
-While we expose certain routes on the sandbox API without the requirement for authentication, we rate-limit these requests aggressively to prevent abuse of the sandbox.  Rate-limiting on authenticated requests
- is far more lenient.
+While we expose certain routes on the sandbox API without the requirement for authentication, we rate-limit these requests significantly to prevent abuse. Rate-limiting on authenticated requests is less restrictive.
 
 ## Installing the Currencycloud SDK
 
 ### 1. Using Maven
 
-To use the Currencycloud SDK in a Maven project, add the following dependency to the project's `pom.xml`:
+To use the Currencycloud SDK in a Maven project, add the following dependency to your project's `pom.xml`:
 
 ```xml
 <dependency>
     <groupId>com.currencycloud.currencycloud-java</groupId>
     <artifactId>currencycloud-java</artifactId>
-    <version>0.7.8</version>
```

Comment:
There is an involved process during release where you have to create a snapshot first - due to the way that MAVEN and Nexus work. See https://github.com/CurrencyCloud/currencycloud-java/wiki/Releasing-the-SDK

Reply:
Added **-SNAPSHOT** to suggested version. Release can only be driven by maintainers

Reply:
One more thing - is there a specific reason for bumping this to 0.9.1 here?  Would 0.7.9 not make more sense as we are adding API functionality but not changing the manner in which the client library works at all?